spec/spec_helper.rb

```diff
@@ -15,6 +15,8 @@
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
 require 'temporal'
+require 'google/protobuf/well_known_types'
```

Comment:
Are we using it outside of the grpc_client specs (where it is already included)?

Reply:
We're using it to generate Google::Protobuf::Timestamps in the fabricators.